workshops/models.py

```diff
@@ -271,26 +288,26 @@ class Person(AbstractBaseUser, PermissionsMixin):
 
     USERNAME_FIELD = 'username'
     REQUIRED_FIELDS = [
-        'personal',
-        'family',
         'email',
-        ]
+    ]
 
     objects = PersonManager()
 
     def get_full_name(self):
         middle = ''
         if self.middle is not None:
             middle = ' {0}'.format(self.middle)
-        return '{0}{1} {2}'.format(self.personal, middle, self.family)
+        return '{0}{1} {2}'.format(self.personal or '', middle, self.family or '')
 
     def get_short_name(self):
-        return self.personal
+        return self.personal or self.username
```

Comment:
Again, I've reverted this back. At some point I thought that `personal` should be optional, but it's not the case.